jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +217,190 @@ def _get_project_status(project_statuses, status):
 
         self.api.user_stories.delete(story_id)
         return migrate_story.id
+
+    def list_all_issues(self, user_id=""):
+        """
+        List all issues for specific user if you didn't pass user_id will list all the issues
+        
+        Args:
+            user_id (int): id of the user.
+
+        Returns:
+            List: List of issues.
+        """
+        return self.api.issues.list(assigned_to=user_id)
+
+    def list_all_projects(self):
+        """
+        List all projects
+        
+        Returns:
+            List: List of projects.
+        """
+        return self.api.projects.list()
+
+    def list_all_milestones(self):
+        """
+        List all milestones
+        
+        Returns:
+            List: List of milestones.
+        """
+        return self.api.milestone.list()
+
+    def list_all_user_stories(self, user_id=""):
+        """
+        List all user stories for specific user if you didn't pass user_id will list all the available user stories
+        
+        Args:
+            user_id (int): id of the user.
+            
+        Returns:
+            List: List of user stories.
+        """
+        return self.api.user_stories.list(assigned_to=user_id)
+
+    def __render_issues_with_details(self, text="Issues", issues=None):
+        """Get issues subject and append them to text
+        
+        Args:
+            text (str): the text that the method will append to it.
+            issues (List): list of all issues that we will get the subject from .
+            
+        Returns:
+            str: string contains the issues subject.
```

Comment:
It's better to export to one of the known formats and to specify this in docstring too, what's the current format of this export? markdown?

IMHO, csv like formats would be a good choice, as this export would be both human and machine readable.

Reply:
Also, it's better to use a template file here if needed (I think won't be needed in case of csv for example).

Reply:
The task was to export the data to markdown files.